zipline/finance/performance/position_tracker.py

```diff
@@ -350,14 +350,16 @@ def get_positions(self):
                     pass
                 continue
 
-            # Note that this will create a position if we don't currently have
-            # an entry
-            position = positions[sid]
+            position = zp.Position(sid)
```

Comment:
Is last_sale_date something that the user should be able to access? It doesn't happen to be in the docs